src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -1434,7 +1415,7 @@ private bool ProcessFile(string filename)
 
                 using (FileStream fs = new FileStream(filename, FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
                 {
-                    using (StreamReader sr = new StreamReader(fs, _textEncoding))
+                    using (StreamReader sr = new StreamReader(fs, Encoding))
```

Comment:
There's an ArgumentNullException path here if Encoding is null.

Reply:
fixed with `[ValidateNotNullOrEmpty]` attribute

Reply:
fixed with attribute